src/Security/RandomGenerator.php

```diff
@@ -13,9 +14,12 @@ class RandomGenerator
     /**
      * @return string A 128-character, randomly generated ASCII string
      * @throws Exception If no suitable CSPRNG is installed
+     * @deprecated 4.4:5.0
```

Comment:
I think these need to be full versions:

```suggestion
     * @deprecated 4.4.0:5.0.0
```